packages/react/src/components/sideNavigation/SideNavigation.stories.tsx

```diff
@@ -0,0 +1,184 @@
+        <div style={{ height: '100%', maxWidth: '400px' }}>{storyFn()}</div>
+      </div>
+    ),
+  ],
+  title: 'Components/SideNavigation',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: { defaultOpenMainLevels: [], toggleButtonLabel: 'Navigate to page', id: 'side-navigation' },
+};
+
+enum ID {
+  MAIN_LEVEL_1 = '#main-level-1',
+  MAIN_LEVEL_2 = '#main-level-2',
+  MAIN_LEVEL_3 = '#main-level-3',
+  MAIN_LEVEL_4 = '#main-level-4',
+  SUB_LEVEL_1 = '#sub-level-1',
+  SUB_LEVEL_2 = '#sub-level-2',
+  SUB_LEVEL_3 = '#sub-level-3',
+  SUB_LEVEL_4 = '#sub-level-4',
+}
+
+const labels = {
+  mainLevel1: 'Main level accordion',
+  mainLevel2: 'Main level accordion',
+  mainLevel3: 'Main level link',
+  mainLevel4: 'Main level external link',
+  subLevel1: 'Sub level link',
+  subLevel2: 'Sub level link',
+  subLevel3: 'Sub level link',
+  subLevel4: 'Sub level link',
+};
+
+const handleClick = (setActive) => (ev) => {
+  ev.preventDefault();
+  setActive(ev.currentTarget.getAttribute('id') as ID);
+};
+
+export const Default = (args) => {
+  const [active, setActive] = React.useState(ID.SUB_LEVEL_1);
+
+  return (
+    <SideNavigation {...args}>
+      <SideNavigation.MainLevel id={ID.MAIN_LEVEL_1} label={labels.mainLevel1}>
+        <SideNavigation.SubLevel
+          active={ID.SUB_LEVEL_1 === active}
+          id={ID.SUB_LEVEL_1}
+          href={ID.SUB_LEVEL_1}
+          label={labels.subLevel1}
+          onClick={handleClick(setActive)}
+        />
+        <SideNavigation.SubLevel
+          active={ID.SUB_LEVEL_2 === active}
+          id={ID.SUB_LEVEL_2}
+          href={ID.SUB_LEVEL_2}
+          label={labels.subLevel2}
+          onClick={handleClick(setActive)}
+        />
+      </SideNavigation.MainLevel>
+      <SideNavigation.MainLevel id={ID.MAIN_LEVEL_2} label={labels.mainLevel2}>
+        <SideNavigation.SubLevel
+          active={ID.SUB_LEVEL_3 === active}
+          id={ID.SUB_LEVEL_3}
+          href={ID.SUB_LEVEL_3}
+          label={labels.subLevel3}
+          onClick={handleClick(setActive)}
+        />
+        <SideNavigation.SubLevel
+          active={ID.SUB_LEVEL_4 === active}
+          id={ID.SUB_LEVEL_4}
+          href={ID.SUB_LEVEL_4}
+          label={labels.subLevel4}
+          onClick={handleClick(setActive)}
+        />
+      </SideNavigation.MainLevel>
+      <SideNavigation.MainLevel
+        active={ID.MAIN_LEVEL_3 === active}
+        id={ID.MAIN_LEVEL_3}
+        href={ID.MAIN_LEVEL_3}
+        label={labels.mainLevel3}
+        onClick={handleClick(setActive)}
+      />
+      <SideNavigation.MainLevel
+        active={ID.MAIN_LEVEL_4 === active}
+        id={ID.MAIN_LEVEL_4}
+        href={ID.MAIN_LEVEL_4}
+        label={labels.mainLevel4}
+        external
+        openInExternalDomainAriaLabel="Opens a different website"
+        onClick={handleClick(setActive)}
+        withDivider
+      />
+    </SideNavigation>
+  );
+};
+
+export const WithIcons = (args) => {
```

Comment:
Should we add one story where the side navigation is used with the normal navigation? It would be cool to see that they fit and work together without problems. Or maybe we will use this combination in the new docs site?

Reply:
Yes, we should have an example of using both, but like Ville said, new docsite should have both navigations and also there is a separate ticket https://helsinkisolutionoffice.atlassian.net/browse/HDS-917 to create pattern documentation for the navigation: what levels to present in which navigation, how to combine those and so on. So I'd say way will put a little more effort to examples of side navigation together with main navigation later on.